File: java/client/src/org/openqa/selenium/devtools/network/model/SearchMatch.java
package org.openqa.selenium.devtools.network.model;

/**
 * Search match for resource
 */
public class SearchMatch {

  /**
   * Line number in resource content
   */
  private Double lineNumber;

  /**
   * Line with match content
   */
  private String lineContent;

  public Double getLineNumber() {
    return lineNumber;
  }

  public void setLineNumber(Double lineNumber) {
    this.lineNumber = lineNumber;
  }

  public String getLineContent() {
    return lineContent;
  }

  public void setLineContent(String lineContent) {
    this.lineContent = lineContent;
  }

}

File: dotnet/src/webdriver/DevTools/AutoGenerated/Debugger/SearchMatch.cs
namespace OpenQA.Selenium.DevTools.Debugger
{
    using Newtonsoft.Json;

    /// <summary>
    /// Search match for resource.
    /// </summary>
    public sealed class SearchMatch
    {
        /// <summary>
        /// Line number in resource content.
        ///</summary>
        [JsonProperty("lineNumber")]
        public double LineNumber
        {
            get;
            set;
        }
        /// <summary>
        /// Line with match content.
        ///</summary>
        [JsonProperty("lineContent")]
        public string LineContent
        {
            get;
            set;
        }
    }
}